src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -91,8 +92,10 @@ def load_command_table(self, command_loader):
 
             command_verb = command_name.split()[-1]
             # To work around http://bugs.python.org/issue9253, we artificially add any new
```

Comment:
I checked http://bugs.python.org/issue9253 mentioned in the comment, but didn't see how it is related to this workaround.